lib/content-store/index.js

```diff
@@ -1,25 +1,33 @@
 const config = require('../../config/config');
 const logger = require('../logger');
 const restHandler = require('./rest-handler');
+const fallbackHandler = require('./fallback-handler');
 const fileHandler = require('./file-handler');
 
 const HANDLER_TYPE = config.contentStore.type;
 
 
 function getHandler(type) {
-  logger.verbose(`Content handler: ${type}`);
-
   if (!type) {
     throw Error('Content handler type is missing. Check `contentStoreType` in config');
   }
 
+  let handler;
+
   switch (type) {
     case 'rest':
-      return restHandler;
+      handler = restHandler;
+      break;
+    case 'fallback':
+      handler = fallbackHandler;
+      break;
     case 'file':
     default:
-      return fileHandler;
+      handler = fileHandler;
```

Comment:
I know it's not strictly part of this PR but I'm not sure I like the default logic here.
If you specify an invalid handler in your config, it would still return the default one instead of erroring which wouldn't be obvious.